src/plugins/whiteboard/whiteboard.yml

```diff
@@ -2,3 +2,39 @@ whiteboard:
   name:
     en: Whiteboard
     de: Whiteboard
+  selectTooltip:
+    en: Select mode
+    de: Select mode
+  freeDrawTooltip:
+    en: Freedraw mode
+    de: Freedraw mode
+  textTooltip:
+    en: Add text
+    de: Add text
+  shapeTooltip:
+    en: Add shape
+    de: Add shape
+  eraseTooltip:
+    en: Erase shape
+    de: Erase shape
```

Comment:
Objekt löschen (maybe also in english "object", because we use shape only for circles etc. not for text for example?)

Reply:
Agreed, changed it.
Added all translations as well, thanks for that!